### ISSUE TITLE
Skill Tree actives listing themselves as a prerequisite

### ISSUE THREAD

**Issue by Marakk** [larsiusprime/tdrpg-bugs#553]
Windows 7
Download
DQ1 HD Beta
Version # 0.2.6

Look at the any character skill tree  and find an unlearned active skill.  If you mouse over this skill, it says "Requires: Character Level XX, 1 point in *name of skill you're currently mousing over*, instead of listing the skill to the left in the skill tree.

Specific example(s): 

Berserker who has yet to learn Swipe - hovering over Swipe gives the tooltip "Requires: Character Level 15, 1 point in Swipe".  Should read "1 point in Madness."

Healer who has yet to learn Holy Light - hovering over Holy Light gives the tooltip "Requires: Character Level 10, 1 point in Holy Light".  Should read "1 point in Blinding".

Ranger who has yet to learn Black Out the Sun - hovering over Black Out the Sun gives the tooltip "Requires: Character Level 20, 1 point in Black Out the Sun".  Should read "1 point in Spread Shot".

Basically, all unlearned active skills in every tree (as far as I can see) list themselves as a prerequisite instead of the immediately preceding skill.

**Comment by larsiusprime** [comment]
Great catch. I was already doing all the logic of finding the required skill, and then accidentally using the localization flag of the original skill.... oops.

Will be fixed in 0.3.2.

**Comment by Marakk** [comment]
Haha, oops, I didn't even notice you were sending out e-mails in regards to beta build bug reports!  My bad.  I'm going to have to download the latest build now and see if any of these crop up for me again, since it seems like you've fixed pretty much every weird thing I was running into.

**Comment by larsiusprime** [comment]
I actually just post on the bug board, which is set up to also email you :) So you can either check the board or your inbox, whichever works.
